src/java/htsjdk/samtools/CRAMIterator.java

```diff
@@ -156,6 +156,9 @@ private void nextContainer() throws IOException, IllegalArgumentException,
             final SAMSequenceRecord sequence = cramHeader.getSamFileHeader()
                     .getSequence(container.sequenceId);
             refs = referenceSource.getReferenceBases(sequence, true);
+            if (refs == null) {
+                throw new RuntimeException(String.format("Contig %s not found in the reference file.", sequence.getSequenceName()));
```

Comment:
Took a second look at the class, and I take it back -- it is using `RuntimeException`, but only for generic errors like "not implemented". `CramFileReader` uses `SAMException` in a couple places, so I think there's precedence for just using that (or a new subclass of that) here.